web/src/elements/forms/SearchSelect/ak-search-select.ts

```diff
@@ -105,12 +106,17 @@ export class SearchSelect<T> extends CustomEmitterElement(AKElement) {
     @state()
     error?: APIErrorTypes;
 
-    static styles = [PFBase, PFForm, PFFormControl, PFSelect];
+    static get styles() {
+        return [PFBase, PFForm, PFFormControl, PFSelect];
+    }
 
     constructor() {
         super();
         if (!document.adoptedStyleSheets.includes(PFDropdown)) {
-            document.adoptedStyleSheets = [...document.adoptedStyleSheets, PFDropdown];
+            document.adoptedStyleSheets = [
+                ...document.adoptedStyleSheets,
+                ensureCSSStyleSheet(PFDropdown),
```

Comment:
_Sigh_.  Blame Storybook.  It wasn't the style sheets in a very specific format.